fin360/fin360.py

```diff
@@ -44,3 +44,16 @@ def authenticate(self, email, password):
         self.access_token = response.json()['access_token']
 
         return AttrDict(response.json())
+
+    def get_transactions_with_details(self, accountid):
+
+        url = "https://www.fin360.in/bank-account/api/v1/transactionsWithDetails/" + accountid + ".json"
```

Comment:
please add a documentation of the response format ie what is returned by the api. An eg transaction would be great